### PR TITLE
Add a message in 'Get Started Page'

### PR DESCRIPTION
**Pull request by chensun**
Add a message to remind users to choose a Tensorflow 2.1 notebook instance.

/assign @IreneGi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3152)

### CODE REVIEW

**Comment by Bobgy** [comment]
Run
npm test -u
And run all tests to update snapshot tests

**Comment by chensun** [comment]
> Run
> npm test -u
> And run all tests to update snapshot tests

Thanks! I was wondering if the snap file is manually updated or maintained by any script.

File: frontend/src/pages/__snapshots__/GettingStarted.test.tsx.snap [hunk 1]
```diff
@@ -5,7 +5,7 @@ exports[`GettingStarted page fallbacks to show pipeline list page if request fai
 - First value
 + Second value
 
-@@ -69,11 +69,13 @@
+@@ -70,11 +70,13 @@
```
Comment (Bobgy) [inline frontend/src/pages/__snapshots__/GettingStarted.test.tsx.snap]: Looks like this is not a good way to do snapshot testing. I will consider alternatives.

**Comment by Bobgy** [comment]
The failure here in xgboost is the same as https://travis-ci.com/kubeflow/pipelines/jobs/290012541.

python2 cannot find enum module, I'm suspecting some modules are no longer supporting python2 because it is no longer maintained.

**Comment by numerology** [comment]
FYI I filed #3155

**Pull request merged**